visualizer/index.html

```diff
@@ -372,8 +372,13 @@
     <div class="externalRules"><div class="content"></div></div>
   </div>
 
-  <div id="contextMenu" hidden><ul></ul></div>
-
+  <div id="node" class="contextMenu" hidden><ul></ul></div>
```

Comment:
Ideally, the `id` should make sense on its own, without knowing the class name. Here, "node" seems like much too generic of a name. What about "parseTreeMenu"?